tests/test_bgp_af.rb

```diff
@@ -28,31 +28,254 @@ def setup
     super
     # Disable and enable feature bgp before each test to ensure we
     # are starting with a clean slate for each test.
-    config('no feature bgp', 'feature bgp')
+    if platform == :nexus
+      config('no feature bgp', 'feature bgp')
+    elsif platform == :ios_xr
+      config('no router bgp')
+      config('no route-policy drop_all')
+    end
   end
 
-  def get_bgp_af_cfg(asn, vrf, af)
-    afi, safi = af
-    string =
-      @device.cmd("show run bgp all | sec 'bgp #{asn}' |  sec 'vrf #{vrf}' | " \
-                  "sec 'address-family #{afi} #{safi}' | no-more")
-    string
+  # rubocop:disable Style/WordArray
+  # rubocop:disable Style/ClassVars
+  # rubocop:disable Metrics/LineLength
```

Comment:
* Style/WordDisable -  I didn't write the inject_map code, so I don't know why the author disabled that one.  Looks about the same to me either way.

* ClassVars - I copied the matrix setup from test_bgp_neighbor_af.rb, which uses class vars.  When RuboCop complained I made them instance vars and then got lots of odd errors when running the tests.  Decided it wasn't worth the trouble and just added them back to being class vars.  Suggestions welcome.